geopy/util.py

```diff
@@ -80,3 +80,10 @@ def decode_page(page):
         else: # requests?
             encoding = page.headers.get("charset") or "utf-8"
             return str(page.content, encoding=encoding)
+
+
+def get_version():
+    import geopy
+    return str(geopy.__version__)
```

Comment:
Hmm, wait, I cannot move geopy.__init__.__version__ as it it used by python distutils/setuptools to determine the version number for packaging. But maybe I find a better way to do it...